tests/prepared_init.rs

```diff
@@ -0,0 +1,33 @@
+use fuel_tx::*;
+use rand::rngs::StdRng;
+use rand::{Rng, SeedableRng};
+
+#[test]
+fn output_prepare_init_zeroes_recipient_and_amount() {
```

Comment:
Since other outputs or parts of the tx may be zeroed in the future, should we clarify this test is only for message outputs?

```suggestion
fn output_prepare_init_zeroes_message_recipient_and_amount() {
```